P202.Training.Data/UserDal.cs

```diff
@@ -0,0 +1,120 @@
+﻿using NHibernate;
```

Comment:
I think we don't need this file anymore right? This logic is now present in `UserRepository`